docker/Dockerfile.k8s

```diff
@@ -0,0 +1,61 @@
+FROM golang:1.12
+WORKDIR /opt/pbm
+COPY . .
+RUN make install
+
+FROM registry.access.redhat.com/ubi7/ubi-minimal
+RUN microdnf update && microdnf clean all
+
+MAINTAINER Percona Development <info@percona.com>
+LABEL name="Percona Backup for MongoDB" \
+      vendor="Percona" \
+      summary="Percona Backup for MongoDB is a distributed, low-impact solution for achieving consistent backups of MongoDB Sharded Clusters and Replica Sets." \
+      description=" This is a tool for creating consistent backups across a MongoDB sharded cluster (or a single replica set), and for restoring those backups to a specific point in time. Percona Backup for MongoDB uses a distributed client/server architecture to perform backup/restore actions."
+
+COPY LICENSE /licenses/
+
+# kubectl needed for Percona Operator for PSMDB
+ENV KUBECTL_VERSION=v1.14.1
+ENV KUBECTL_MD5SUM=223668b6d47121a9011645b04f5ef349
+RUN curl -o /usr/bin/kubectl \
```

Comment:
I would recommend adding -f to all curl invocations so that they fail on non-success HTTP codes